applications/luci-app-ntpc/Makefile

```diff
@@ -7,7 +7,8 @@
 include $(TOPDIR)/rules.mk
 
 LUCI_TITLE:=NTP time synchronisation configuration module
-LUCI_DEPENDS:=+luci-base +luci-compat +ntpclient
+LUCI_DESCRIPTION:=LuCI support for configuring the ntpclient package
```

Comment:
@systemcrash do you think we need a better description than that or should it suffice?

Reply:
Golden. Done I think.